README.md

```diff
@@ -94,16 +94,16 @@ for all unhandled checks
     unhandled_checks = client.service_status(params={'filter': 'unhandled'})['list']
 
 for adding an acknowledgement to the unhandled event for Opsview on check Connectivity - LAN
-    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview', 'svc.servicename': 'Connectivity - LAN'} , data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
+    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview'}, 'svc.servicename': 'Connectivity - LAN'} , data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
 
 for adding an acknowldegement to the unhandled host event for Opsview
-    acknowldege_details = client.acknowledge_event(params={'hst.hostname': 'Opsview', data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
+    acknowldege_details = client.acknowledge_event(params={'hst.hostname': 'Opsview'}, data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
 
 for adding an acknowldegement to all unhandled host and service events for Opsview
-    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview', data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
+    acknowldege_details = client.acknowledge_event(params={'svc.hostname': 'Opsview'}, data={'notify': str(int(bool(True))), 'sticky': str(int(bool(True))), 'comment': 'Acknowledged and actioned by user'})
 
 for deleting the acknowledgement for Opsview against check Connectivity - LAN
-    acknowledge_details = client.acknowledge_delete(params={'svc.hostname': 'Opsview', 'svc.servicename': 'Connectivity - LAN'})
+    acknowledge_details = client.acknowledge_delete(params={'svc.hostname': 'Opsview'}, 'svc.servicename': 'Connectivity - LAN'})
```

Comment:
This doesn't look right - there are unbalanced `}` on the line

Reply:
Ah, I think there are multiple commits all touching the same thing

Reply:
Hi Duncan,
Yes, sorry about that,  I didn't see my mistake in the readme until I had
committed it.
Am happy to delete the pull request and resubmit it in a single commit.
Just let me know if this is wanted

Denise